x-pack/platform/plugins/shared/fleet/server/plugin.ts

```diff
@@ -816,16 +816,7 @@ export class FleetPlugin
         core.elasticsearch.client.asInternalUser,
         internalSoClient
       ),
-      agentPolicyService: {
-        get: agentPolicyService.get,
-        list: agentPolicyService.list,
-        getFullAgentPolicy: agentPolicyService.getFullAgentPolicy,
-        getByIds: agentPolicyService.getByIDs,
-        turnOffAgentTamperProtections:
-          agentPolicyService.turnOffAgentTamperProtections.bind(agentPolicyService),
-        fetchAllAgentPolicies: agentPolicyService.fetchAllAgentPolicies,
-        fetchAllAgentPolicyIds: agentPolicyService.fetchAllAgentPolicyIds,
-      },
+      agentPolicyService,
```

Comment:
This was done due to the fact that I needed a `create` method that depends on a lot of other private/internal methods.

I had to either make the methods public + add here; or I could pass the service itself. Potentially there might be other way, but I'm not familiar enough with Kibana development yet to know, please tell me if there's a better way :)